### PR TITLE
Fix editor in docs

### PR DESCRIPTION
**Pull request by tmarshall**
Followup to https://github.com/instacart/Snacks/pull/323

Fixes editor for code snippits.

### CODE REVIEW

File: docs/colors.md [hunk 1]
```diff
@@ -39,7 +40,7 @@ const colorsSwatches = Object.keys(colors).map(name => {
   )
 })
 
-<ul style={wrapperStyles}>
- { colorsSwatches }
-</ul>
+; <ul style={wrapperStyles}>
```
Comment (tmarshall) [inline docs/colors.md]: must start any render with `;` or else styleguidist will throw.

File: styleguide.config.js [hunk 2]
```diff
@@ -1,4 +1,10 @@
 const path = require('path');
+
+const webpackConfig = require('./webpack.config');
+webpackConfig.resolve = webpackConfig.resolve || {};
+webpackConfig.resolve.alias = webpackConfig.resolve.alias || {};
+webpackConfig.resolve.alias['ic-snacks'] = path.join(__dirname, 'dist', 'snacks.js');
```
Comment (tmarshall) [inline styleguide.config.js]: resolving `'ic-snacks'` to built package. Makes readmes more readable & sensical

File: src/index.js [hunk 3]
```diff
@@ -84,6 +85,7 @@ export {
   Button,
   CircleButton,
   Radio,
+  RadioGroup,
```
Comment (tmarshall) [inline src/index.js]: `RadioGroup` wasn't exported before?

Comment (loganmurdock) [reply src/index.js]: Really? Huh, I guess we don't have it anywhere in `customers`

**Pull request merged**